cypress/elements/optionsModal/axes.js

```diff
@@ -17,6 +19,12 @@ export const setAxisTitleText = (axis, text) =>
 export const setAxisTitleToCustom = () =>
     cy.getBySel(axisTitleRadiosEl).contains('Custom').click()
 
+export const setAxisTitleToAuto = () =>
```

Comment:
These 3 functions could be made into a single function `setAxisTitleTo(option)` or maybe `setAxisTitleOptionTo(option)`. In the test it would look like:

`setAxisTitleOptionTo('None')`

which one could argue is more readable than

`setAxisTitleOptionToNone()`

Just a thought. It's your call.

Reply:
Good idea 👍